frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -26,6 +27,10 @@ import {
 } from '../../types';
 import NetworkTypeOptions from './NetworkTypeOptions';
 
+const hasHyperConvergeCRD = !!['v1beta1'].find(
```

Comment:
```suggestion
const hasHyperConvergeCRD = () =>  !!['v1beta1', 'v1alpha1'].find(
```
we need to support also `v1alpha1`, and I think it will be nicer if it's a method that is calculated when needed